File: Python/URI_judge_1012.py
p1 = input().split()
a, b, c = p1
pi = 3.14159

tr = (float(a) * float(c))/2
cr = pi * (float(c)* float(c))
t = float(c) *(float(a) + float(b)) / 2
q = float(b) * float(b)
r = float(a) * float(b)

print(f"TRIANGULO: {tr:.3f}")
print(f"CIRCULO: {cr:.3f}")
print(f"TRAPEZIO: {t:.3f}")
print(f"QUADRADO: {q:.3f}")
print(f"RETANGULO: {r:.3f}")

File: Ruby/URI_judge_1012.rb
X = gets.split(' ')

TRI = (X[0].to_f * X[2].to_f) / 2
C = 3.14159 * (X[2].to_f ** 2)
TRA = ((X[0].to_f + X[1].to_f) * X[2].to_f) / 2
Q =  X[1].to_f ** 2
R = X[0].to_f * X[1].to_f

puts "TRIANGULO: %.3f" % TRI
puts "CIRCULO: %.3f" % C
puts "TRAPEZIO: %.3f" % TRA
puts "QUADRADO: %.3f" % Q
puts "RETANGULO: %.3f" % R